lib/exceptional_synchrony/event_machine_proxy.rb

```diff
@@ -5,6 +5,8 @@
 require 'em-synchrony/em-http'
 
 module ExceptionalSynchrony
+  class FatalRunError < Exception; end
```

Comment:
Can you put a comment above here about how it's important to not inherit from `StandardError`?